### PR TITLE
finding subtree size of each node  

### PR DESCRIPTION
**Pull request by Tanya933**
issue #1226  fixed . @tarun26091999 @rudrakshi99 @plazzy99 @nakul-19  Please review the PR .Thankyou

### CODE REVIEW

File: Code/C++/subtree_size.cpp [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    for (int i = 0; i < v; i++) {
+        cout << "node :" << i << "  subtree_nodes : " << subtree_totalNodes[i] << "\n";
+    }
+    return 0;
+}
+
+
+
+/*USER'S INPUT AND OUTPUT
+
```
Comment (krishna-NIT) [inline Code/C++/subtree_size.cpp:97]: Mention Space And Time Complexity Here

Comment (Tanya933) [reply Code/C++/subtree_size.cpp:97]: okay working on it.

**Comment by krishna-NIT** [comment]
@Tanya933  Delete the file that You have added to this location (Code/C++/a.out)
Update Readme of Related problem, if you got confused about where to update just
Refer to any merged pr and follow that. Just add your issue name with its location at end of the readme in questions Section

**Comment by Tanya933** [comment]
okay @krishna-NIT

**Comment by krishna-NIT** [comment]
I Think you haven't pushed your changes

**Comment by krishna-NIT** [comment]
> I Think you haven't pushed your changes

@Tanya933  Check it once

**Comment by Tanya933** [comment]
@krishna-NIT  Please review the PR.Thankyou

**Comment by krishna-NIT** [comment]
@rish-singhal LGTM

**Comment by Tanya933** [comment]
issue #1226 fixed . @tarun26091999 @rudrakshi99 @plazzy99 @nakul-19 @rish-singhal  .Please merge this issue .Thankyou

File: Code/C++/subtree_size.cpp (hunk 1 above)
Comment (tarun26091999) [reply Code/C++/subtree_size.cpp:97]: complexities have to come at bottom.

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
remove the example from top.

**Comment by tarun26091999** [comment]
also resolve the conflict.

**Comment by Tanya933** [comment]
@tarun26091999 .Removed example from top ,moved the complexity at the bottom and resolved the conflicts.Please review .Thankyou

**Comment by rish-singhal** [comment]
Resolve conflicts

**Pull request closed without merging**